packages/runtime/test/src/logicsig.ts

```diff
@@ -0,0 +1,95 @@
+import { decodeAddress, multisigAddress } from "algosdk";
+import { assert } from "chai";
+
+import { StoreAccount } from "../../src/account";
+import { Runtime } from "../../src/runtime";
+import { getProgram } from "../helpers/files";
+import { useFixture } from "../helpers/integration";
+
+const programName = "escrow.teal";
+const multiSigProg = "sample-asc.teal";
+
+describe("Logic Signature Test", () => {
+  useFixture("escrow-account");
+  const john = new StoreAccount(10);
+  const bob = new StoreAccount(100);
+  const runtime = new Runtime([john, bob]);
+  const johnPk = decodeAddress(john.address).publicKey;
+
+  it("should sign the lsig by john(delegated signature)", () => {
+    const lsig = runtime.getLogicSig(getProgram(programName), []);
+
+    lsig.sign(john.account.sk);
+    const result = lsig.verify(johnPk);
+
+    assert.equal(result, true);
```

Comment:
```suggestion
    assert.isTrue(lsig.verify(johnPk));
```

Reply:
We can do similar updates below (isTrue, isFalse...)

https://www.chaijs.com/api/assert/#method_istrue